### PR TITLE
feat: support dark/light color themes

### PR DESCRIPTION
**Pull request by antfu**
Support passing an object to `theme` and display the theme based on the color mode.

```ts
theme: {
  dark: 'min-dark',
  light: 'min-light'
}
```

Also extract the icon URLs into variables for centralization. 

## Screenshots

image
image

### CODE REVIEW

File: src/node/markdown/plugins/highlight.ts [hunk 1]
```diff
@@ -1,14 +1,28 @@
-import escapeHtml from 'escape-html'
 import { getHighlighter } from 'shiki'
+import type { ThemeOptions } from '../markdown'
 
-export const highlight = async (theme = 'material-palenight') => {
-  const highlighter = await getHighlighter({ theme })
+export async function highlight(
+  theme: ThemeOptions = { light: 'vitesse-light', dark: 'vitesse-dark' }
```
Comment (antfu) [inline src/node/markdown/plugins/highlight.ts:5]: By no means to self prompt :P 

Was trying to demonstrate this feature, but `material-palenight` does not seem to have a light mode pair.

Feel free to change the themes that make sense to you: https://github.com/shikijs/shiki/tree/main/packages/shiki/themes

File: src/client/theme-default/styles/vars.css [hunk 2]
```diff
@@ -183,6 +183,15 @@
   --vp-layout-max-width: 1440px;
 }
 
+/**
+ * Icons
+ * -------------------------------------------------------------------------- */
+
+:root {
+  --vp-icon-copy: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' height='20' width='20' stroke='rgba(128,128,128,1)' stroke-width='2' class='h-6 w-6' viewBox='0 0 24 24'%3E%3Cpath stroke-linecap='round' stroke-linejoin='round' d='M9 5H7a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V7a2 2 0 0 0-2-2h-2M9 5a2 2 0 0 0 2 2h2a2 2 0 0 0 2-2M9 5a2 2 0 0 1 2-2h2a2 2 0 0 1 2 2'/%3E%3C/svg%3E");
+  --vp-icon-coied: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' height='20' width='20' stroke='rgba(128,128,128,1)' stroke-width='2' class='h-6 w-6' viewBox='0 0 24 24'%3E%3Cpath stroke-linecap='round' stroke-linejoin='round' d='M9 5H7a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V7a2 2 0 0 0-2-2h-2M9 5a2 2 0 0 0 2 2h2a2 2 0 0 0 2-2M9 5a2 2 0 0 1 2-2h2a2 2 0 0 1 2 2m-6 9 2 2 4-4'/%3E%3C/svg%3E");
```
Comment (Shinigami92) [inline src/client/theme-default/styles/vars.css:192]: did you mean `copied`?

Comment (antfu) [reply src/client/theme-default/styles/vars.css]: Nice catch! Fixed

**Review by kiaking — APPROVED** [review]
Thanks a bunch! 🙌 

I switched back to palenight schema because that's what Vue.js website is using, and that was due to Evan's survay result on Twitter a long time ago.

It might not be the same for VitePress. We could do anotehr survay before shipping 1.0.0 💪 

Also added anotehr variable for customizing "copy code active text".

**Pull request merged**